boilerplate/package.json

```diff
@@ -32,39 +30,39 @@
   },
   "dependencies": {
     "@expo-google-fonts/space-grotesk": "^0.2.2",
-    "@expo/metro-runtime": "~3.2.1",
+    "@expo/metro-runtime": "~4.0.0",
     "@react-navigation/bottom-tabs": "^6.3.2",
     "@react-navigation/native": "^6.0.2",
     "@react-navigation/native-stack": "^6.0.2",
-    "@shopify/flash-list": "^1.6.4",
+    "@shopify/flash-list": "1.7.1",
     "apisauce": "3.0.1",
     "date-fns": "^4.1.0",
-    "expo": "~51.0.8",
-    "expo-application": "~5.9.1",
-    "expo-build-properties": "~0.12.1",
-    "expo-dev-client": "~4.0.14",
-    "expo-font": "~12.0.5",
-    "expo-linking": "~6.3.1",
-    "expo-localization": "~15.0.3",
-    "expo-splash-screen": "~0.27.4",
-    "expo-status-bar": "~1.12.1",
-    "expo-system-ui": "~3.0.7",
+    "expo": "^52.0.0",
+    "expo-application": "~6.0.1",
+    "expo-build-properties": "~0.13.1",
+    "expo-dev-client": "~5.0.1",
+    "expo-font": "~13.0.1",
+    "expo-linking": "~7.0.2",
+    "expo-localization": "~16.0.0",
+    "expo-splash-screen": "~0.29.7",
+    "expo-status-bar": "~2.0.0",
+    "expo-system-ui": "~4.0.2",
     "i18next": "^23.14.0",
     "intl-pluralrules": "^2.0.1",
     "mobx": "6.10.2",
     "mobx-react-lite": "4.0.5",
     "mobx-state-tree": "5.3.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
     "react-i18next": "^15.0.1",
-    "react-native": "0.74.5",
-    "react-native-drawer-layout": "4.0.0-alpha.9",
-    "react-native-gesture-handler": "~2.16.1",
+    "react-native": "0.76.1",
+    "react-native-drawer-layout": "^4.0.1",
+    "react-native-gesture-handler": "~2.20.2",
     "react-native-keyboard-controller": "^1.12.7",
-    "react-native-mmkv": "^2.12.2",
-    "react-native-reanimated": "~3.15.0",
-    "react-native-safe-area-context": "4.10.5",
-    "react-native-screens": "3.31.1",
+    "react-native-mmkv": "^3.1.0",
```

Comment:
react-native-mmkv v3 requires newArchEnabled to be true

Reply:
good catch thanks!